Readme.md

```diff
@@ -246,6 +246,15 @@ Status.review_pending.description              # => 'Some description...'
 Status.draft.description                       # => nil
 ```
 
+For every attribute, you have predicate methods for checking presence of each attribute:
```

Comment:
Would probably elaborate that it is actually calling `present?` on the attribute